components/ledger/cmd/storage_test.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_StorageCommands(t *testing.T) {
```

Comment:
Check errors.

Reply:
All errors are now checked. As you can see, the NewStorageDelete command is returning an error, which I'm not sure why. Do you know?

Reply:
What is strange is that, it returns an error when running tests for this package only, but none when running all the packages at once.